Gems/Multiplayer/Code/Source/Components/LocalPredictionPlayerInputComponent.cpp

```diff
@@ -26,12 +26,14 @@ namespace Multiplayer
     AZ_CVAR(uint32_t, cl_PredictiveStateHistorySize, 120, nullptr, AZ::ConsoleFunctorFlags::Null, "Controls how many inputs of predictive state should be retained for debugging desyncs");
 #endif
 
+#if AZ_TRAIT_SERVER
```

Comment:
Thoughts on adding/using AZ_CLIENT_CVAR/AZ_SERVER_CVAR instead of #if AZ_TRAIT_SERVER/#if AZ_CLIENT_SERVER?

Reply:
Are you suggesting a runtime cvar? We definitely want this to be a compile time determination.

Reply:
Yeah, removing server only code from client binaries is a critical must-need for MMOs, free to play titles, and anything else where preventing 3rd parties standing up grey servers is a business critical requirement.